theano/sandbox/cuda/tests/test_opt.py

```diff
@@ -56,6 +56,20 @@ def test_local_assert():
     assert isinstance(a_op[0].inputs[0].type, CudaNdarrayType)
 
 
+def test_local_assert_no_cpu_op():
+    numpy.random.seed(1)
+    m = np.random.uniform(-1, 1, (10, 10)).astype("float32")
+    ms = theano.shared(m, name="m_shared")
+    z = ms**2 + 3
+    mode_local_assert = mode.including("local_assert_no_cpu_op")
```

Comment:
mode do not exist here. You must get it:

mode=theano.compile.mode.get_default_mode()